cmd/loki/Dockerfile

```diff
@@ -2,6 +2,18 @@ FROM golang:1.17.2 as build
 
 COPY . /src/loki
 WORKDIR /src/loki
+
+# Copying this from loki-build-image/Dockerfile
+# because these dependencies are required by the Makefile in certain
+# circumstances. This should be removed and the base image above changed
+# from golang to loki-build-image once the latter becomes available for arm.
+RUN GO111MODULE=on go install \
```

Comment:
I guess you want to add this to the build-image not the actual Loki image no ?

Reply:
If you're referring to the loki-build-image, its already part of that.

The reason that I want to add it here, is because `make BUILD_IN_CONTAINER=false loki` below in this same file uses it. And given that BUILD_IN_CONTAINER=false here, goyacc is not found.

Reply:
As shown here:
https://github.com/grafana/loki/blob/da9b4c9369d1221844c2fedac4b16096e7e44ec6/Makefile#L317

goyacc is used directly because of `BUILD_IN_CONTAINER=false`